tests/installation/first_boot.pm

```diff
@@ -22,19 +22,8 @@ sub handle_login {
     if (get_var('DM_NEEDS_USERNAME')) {
         type_string $username;
     }
-    if (match_has_tag("sddm")) {
-        # make sure choose plasma5 session
-        assert_and_click "sddm-sessions-list";
```

Comment:
the pam auth is probably from send_key "ret"; in line 28, before assert_screen "displaymanager-password-prompt"; sddm has the user pre-selected and the first active input box is the password prompt.

The various DMs in overview:
* xdm: requires typing username & password
* sddm: user is pre-selected: only type password (initial input field)
* gdm: requires to 'select the user' from the list (ret does that) then type password
* lightdm: user is pre-selected: only password (initial input field) needs to be typed (there is only the other_DE tests triggering it, they do the login sequence from withing wait_boot - where the same 'sddm hack' still exists